pkg/cmd/roachtest/tests/restore.go

```diff
@@ -813,6 +813,58 @@ func (tpce tpceRestore) DatabaseName() string {
 	return "tpce"
 }
 
+type tpccRestoreOptions struct {
+	warehouses     int
+	workers        int
+	waitFraction   float64
+	queryTraceFile string
+}
+
+type tpccRestore struct {
+	opts tpccRestoreOptions
+}
+
+func (tpccRestore) getNodes(c cluster.Cluster) (crdbNodes, workloadNode option.NodeListOption) {
+	return c.Range(1, c.Spec().NodeCount-1), c.Node(c.Spec().NodeCount)
```

Comment:
why is this implemented by the tpcc workload? note we already have a `hardwareSpecs.getCRDBNodes()` func

Reply:
Oh whoops didn't realize we had an implementation already